### PR TITLE
Deep Maints Mega Update

### PR DESCRIPTION
**Pull request by Kazkin**
Check the individual pushes for contents.

### CODE REVIEW

File: code/modules/mob/living/carbon/superior_animal/psi_monsters/ai.dm [hunk 1]
```diff
@@ -4,7 +4,14 @@
 		if(last_noise <= (world.time) && aggro_noise)
 			playsound(src.loc, list(aggro_noise), 120, 1)
 			last_noise = world.time + 30 SECONDS
-		alpha = 255
+		animate(src, alpha = chameleon_skill, time = 2)
+		sleep(2)
```
Comment (Trilbyspaceclone) [inline code/modules/mob/living/carbon/superior_animal/psi_monsters/ai.dm:8]: Its less performance heavy to do spawn(2)
iirc

File: code/modules/mob/living/carbon/superior_animal/psi_monsters/ai.dm [hunk 2]
```diff
@@ -4,7 +4,14 @@
...
+		animate(src, alpha = 55, time = 2)
+		sleep(2)
+		animate(src, alpha = 155, time = 2)
+		sleep(2)
+		animate(src, alpha = 255, time = 2)
+		sleep(2)
```
Comment (Trilbyspaceclone) [inline code/modules/mob/living/carbon/superior_animal/psi_monsters/ai.dm:14]: This has no use?

File: code/modules/mob/living/carbon/superior_animal/psi_monsters/ai.dm (hunk 1 above)
Comment (r4d6) [reply code/modules/mob/living/carbon/superior_animal/psi_monsters/ai.dm:8]: sleep() is used to pause the proc itself. spawn() doesn't pause the proc.

File: code/modules/mob/living/carbon/superior_animal/psi_monsters/ai.dm (hunk 2 above)
Comment (r4d6) [reply code/modules/mob/living/carbon/superior_animal/psi_monsters/ai.dm:14]: It is for a gradual fade-in/out.

Comment (Trilbyspaceclone) [reply code/modules/mob/living/carbon/superior_animal/psi_monsters/ai.dm:14]: why have it sleep at the end?

**Pull request merged**